app/Http/ViewHelpers/Adminland/AdminSoftwareViewHelper.php

```diff
@@ -132,16 +132,15 @@ public static function seats($employees, Company $company): Collection
      *
      * @param Software $software
      * @param Company $company
-     * @param string $criteria
+     * @param string|null $criteria
      * @return Collection
      */
-    public static function getPotentialEmployees(Software $software, Company $company, string $criteria): Collection
+    public static function getPotentialEmployees(Software $software, Company $company, ?string $criteria): Collection
     {
         // get list of employees who have the software
         $employees = $software->employees()
             ->select('id')
-            ->pluck('id')
-            ->toArray();
+            ->pluck('id');
```

Comment:
Are you sure? `pluck` returns a collection, not an array.

Reply:
Laravel will convert it to an array when required.